rancher/distros-test-framework
# Add test for k3s Traefik IngressRoute

#### Proposed Changes ####


This adds a test for Traefik CRDs. It should be able to catch regressions such as https://github.com/k3s-io/k3s/issues/8755 in the future.



#### Types of Changes ####


Issue validation and regression enhancement

#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
Jenkins jobs available to review upon request

2. Verify code lint; we should not have errors.


3. Update the documentation if needed.


4. Update makefile and docker run if adding new tests.


5. Run your tests at least 4 times with all configurations needed and possible.


6. If needed test with different os types.


#### Linked Issues ####


https://github.com/k3s-io/k3s/issues/8755

#### Further Comments ####
